Ginger/GingerCoreCommon/Actions/ActLogAction.cs

```diff
@@ -88,9 +98,29 @@ public override String ActionType
 
         public override eImageType Image { get { return eImageType.Empty; } }
 
+        eLogLevel GetLogLevel(LogLevel loglevel)
+        {
+            eLogLevel logLevel;
+            switch(loglevel)
+            {
+                case LogLevel.ERROR:
+                    logLevel = eLogLevel.ERROR;
+                    break;
+                case LogLevel.INFO:
+                    logLevel = eLogLevel.INFO;
+                    break;
+                case LogLevel.WARN:
+                    logLevel = eLogLevel.WARN;
+                    break;
+                default:
+                    throw new KeyNotFoundException();
+            }
+            return logLevel;
+        }
+
         public override void Execute()
-        {            
-            Reporter.ToLog(SelectedLogLevel, GetInputParamCalculatedValue("LogText"));            
+        {         
+            Reporter.ToLog(GetLogLevel(SelectedLogLevel), GetInputParamCalculatedValue("LogText"));            
```

Comment:
how come PR not includes also the ActionPage changes with new bindings to new enum?

Reply:
just changed SelectedLogLevel datatype to new enum, so it will pick new enum values.